src/pages/management/configuration-api/index.mdx

```diff
@@ -2125,10 +2127,12 @@ For properties without an explicit value assignment, the method will return the
 |                      |                                                                                 |
 | -------------------- | ------------------------------------------------------------------------------- |
 | Method URL           | `https://api.livechatinc.com/v3.4/configuration/action/list_license_properties` |
-| Required scopes__*__ | -                                                                               |
+| Required scopes      | `properties.license.value--my:rw`*, `properties.license.value--all:rw`**        |
```

Comment:
why do we require `rw` when the method just lists properties?

Reply:
Replaced with `ro`.